compiler/rustc_error_codes/src/error_codes/E0791.md

```diff
@@ -0,0 +1,43 @@
+An unaligned references to a field of a [packed] struct got created.
+
+[packed]: https://doc.rust-lang.org/reference/type-layout.html#the-alignment-modifiers
+
+### Example
```

Comment:
Please follow the format of the other error codes.

Reply:
FWIW https://github.com/rust-lang/rust/blob/03f6bb63669d099c57808628b226716ab572e22e/compiler/rustc_error_codes/src/error_codes.rs#L8 links to a closed (not merged) PR with a defunct 'rendered' link. I don't know what the format is.

Reply:
I didn't even know there was a RFC. Normally the format is enforced through some checks (which isn't great). For example in [this one](https://github.com/rust-lang/rust/blob/master/compiler/rustc_error_codes/src/error_codes/E0069.md):

```
Small sum-up of the error.

Erroneous code example:

[code example]

Explanation of what went wrong.

[fixed code example (if applicable)]

Extra information (like a link to the Rust Book for example)
```

Reply:
Should this link to https://rust-lang.github.io/rfcs/1567-long-error-codes-explanation-normalization.html instead ?

Reply:
Yes that would be a huge improvement!

Reply:
> I didn't even know there was a RFC.

You are the author of that PR...?

Anyway I will update the link and also the error description. Just waiting for the try build to complete.

Reply:
... Don't trust my memory.

Thanks for sending the fix.

Reply:
@GuillaumeGomez please take a look at the new error code page.